tests/protocols/test_websocket.py

```diff
@@ -9,7 +9,7 @@
 import websockets
 import websockets.client
 import websockets.exceptions
-from typing_extensions import TypedDict
+from typing import TypedDict
```

Comment:
I think you need to use `typing.TypedDict` instead of changing the import.

Reply:
Ah I see, typing is already imported further up.  Will adjust.